exporter/metric/metric.go

```diff
@@ -217,34 +213,18 @@ func (me *metricExporter) createMetricDescriptorIfNeeded(ctx context.Context, md
 func (me *metricExporter) exportTimeSeries(ctx context.Context, rm metricdata.ResourceMetrics) error {
 	tss := []*monitoringpb.TimeSeries{}
 	mr := me.resourceToMonitoredResourcepb(rm.Resource)
-	var errs []error
+	var aggError error
 
 	for _, scope := range rm.ScopeMetrics {
 		for _, metrics := range scope.Metrics {
 			ts, err := me.recordToTspb(metrics, mr, scope.Scope)
-			if err != nil {
-				errs = append(errs, err)
-			} else {
```

Comment:
this else prevented appending timeseries if recordToTspb returned a partial error (an error, and some timeseries)